api/swagger.json

```diff
@@ -0,0 +1,711 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "description": "This is the API specification for Chapter",
+    "version": "1.0.0",
+    "title": "Chapter",
+    "termsOfService": "http://chapter.io/terms",
+    "contact": {
+      "email": "author@chapter.io"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "tags": [
+    {
+      "name": "location",
+      "description": "Everything about your Locations",
+      "externalDocs": {
+        "description": "Find out more",
+        "url": "http://chapter.io"
+      }
+    },
+    {
+      "name": "chapter",
+      "description": "Everything about your Chapters",
+      "externalDocs": {
+        "description": "Find out more",
+        "url": "http://chapter.io"
+      }
+    },
+    {
+      "name": "user",
+      "description": "Everything about your Users",
+      "externalDocs": {
+        "description": "Find out more",
+        "url": "http://chapter.io"
+      }
+    }
+  ],
+  "paths": {
+    "/locations": {
+      "post": {
+        "tags": [
+          "location"
+        ],
+        "summary": "Add a new location to the map",
+        "description": "",
+        "operationId": "addLocation",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/Location"
+              }
+            }
+          },
+          "description": "Location coordinates",
+          "required": true
+        },
+        "responses": {
+          "405": {
+            "description": "Invalid input"
```

Comment:
405 is Method Not Allowed, I'd suggest updating the description to align with the HTTP reason for 405's

Reply:
And if a different status code is used to invalid input then document that here

Reply:
Fair point - I'll fix this. If you have the bandwidth, I could use the help as well. As is obvious, this was created in a hurry 😄